instat/dlgDescribeTwoVarGraph.Designer.vb

```diff
@@ -34,9 +34,9 @@ Partial Class dlgDescribeTwoVarGraph
         '
         'cmdOptions
         '
-        Me.cmdOptions.Location = New System.Drawing.Point(10, 197)
+        Me.cmdOptions.Location = New System.Drawing.Point(311, 228)
         Me.cmdOptions.Name = "cmdOptions"

```

Comment:
Yeah, this is a good point. I don't know where we want it. It gets put in both places in different dialogs, I just want consistency and I can't decide which place is better.
I feel it gets a little "lost" in the place it was in, however, most plot dialogs do have it there!
You can change it back if you want, sorry!